docs/button.rst

```diff
@@ -0,0 +1,36 @@
+Buttons
+*******
+
+.. py::module:: microbit
+
+There are two buttons on the board, called ``button_a`` and ``button_b``.
+
+
+Classes
```

Comment:
But then we would have some repetition with the same method described on both instances of the Button class.
Perhaps we could just mention in the class description, that it's not actually available?